subsys/bluetooth/shell/gatt.c

```diff
@@ -340,7 +340,8 @@ static int cmd_mread(const struct shell *shell, size_t argc, char *argv[])
 
 	read_params.func = read_func;
 	read_params.handle_count = i;
-	read_params.handles = h; /* not used in read func */
+	read_params.multi.handles = h; /* not used in read func */
```

Comment:
What does the comment mean?

Reply:
I'm not sure, as you noticed this commit changes API, and this was obviously not compiling and I had to change it, I left the comment as it was before, as changing this comment is not related to the issue